### DISCUSSION TITLE
Documentação da Svelte em Português

### DISCUSSION THREAD

**Discussion by nazarepiedady** [frontendbr/forum#2433] [Mostre seu trabalho]
Olá caros colegas, espero que todos estejam bem de saúde e que Deus abençoe ricamente vossas vidas, eu gostaria de partilhar este novo projeto em que estou a trabalhar que é a tradução da documentação oficial da Svelte para o Português, eu tenho por habito de traduzir material das tecnologias que estou a estudar e ajudar assim a pavimentar o caminho para os que vierem a estudar os mesmos assuntos para que o passam fazer com mais conforto e facilidade.

A documentação está a ser servida através da plataforma da Vercel no endereço: https://svelte-docs-pt.vercel.app

E código-fonte do mesmo está no endereço: https://github.com/nazarepiedady/svelte-docs-pt

Não esqueça de doar as vossas estrelas ao projeto e de partilhar a noticia com os vossos conhecidos.

**Comment by ninetails** [comment]
Hoje mesmo estava vendo sobre `pnpm` e vi que também está com a documentação em aberto em:

> https://crowdin.com/project/pnpm/pt-BR

Só aproveitando a thread :D

**Reply by nazarepiedady** [reply]
@ninetails estes não são os únicos projetos neste sentido, existe mais, mas a comunidade quase não ajuda estes projetos a crescerem, e muito menos ajuda a dar visibilidade para os mesmos.
